Dockerfile

```diff
@@ -1,19 +1,28 @@
-FROM alpine
+FROM debian:11 as builder
+ARG PDNS_VERSION=4.7.3
+
+WORKDIR /build
+RUN apt update && \
+    apt install -y curl bzip2 g++ python3-venv libtool make pkg-config \
+    libboost-all-dev libssl-dev libluajit-5.1-dev libcurl4-openssl-dev libsqlite3-dev
+RUN curl -sL https://downloads.powerdns.com/releases/pdns-$PDNS_VERSION.tar.bz2 | tar -jx
+WORKDIR /build/pdns-$PDNS_VERSION
+RUN ./configure --with-modules='bind gsqlite3' && \
+    make -j $(nproc) && \
+    make install
+
+FROM debian:11-slim
+
+RUN apt update && apt install -y curl sqlite3 luajit && apt clean
 
 ARG BOOTSTRAP_SQL=schema.sqlite3-4.5.x.sql
 
 ADD entrypoint.sh /entrypoint/script
 ADD $BOOTSTRAP_SQL /entrypoint/bootstrap.sql
```

Comment:
If you are compiling from source, you must have Somewhere :tm: the sql instead of using the one at the root of the repository.

You supersede my previous PR (#11) but it is still a good point where to inspire yourself for this: https://github.com/txqueuelen/pdns-stateless/pull/11/files#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557R4

Reply:
You're completely right, I can source that from the tarball. Done!